lib/ansible/modules/windows/setup.ps1

```diff
@@ -318,11 +323,12 @@ if($gather_subset.Contains('platform')) {
 
     $ansible_facts += @{
         ansible_architecture = $win32_os.OSArchitecture
-        ansible_domain = $ip_props.DomainName
-        ansible_fqdn = ($ip_props.Hostname + "." + $ip_props.DomainName)
-        ansible_hostname = $env:COMPUTERNAME
+        ansible_domain = $domainSuffix
```

Comment:
You are using the value which may have `.` at the start, currently we just return the domain part and not the leading `.`.